exercises/allergies/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "allergies.h"
+#include <stdlib.h>
+
+static const int scores[] = {
+   1,
+   2,
+   4,
+   8,
+   16,
+   32,
+   64,
+   128
+};
+
+bool is_allergic_to(Allergen_t allergen, int score)
+{
+   return ((score & scores[allergen]) == scores[allergen]);
+}
+
+void get_allergens(int score, Allergen_List_t * list)
+{
+   list->allergens = calloc(0, sizeof(Allergen_t) * Allergen_Last);
```

Comment:
Additionally, the signature for `calloc` is `calloc(<number of items>, <item size>)` so I think this should be `calloc(Allergen_Last + 1, sizeof(Allergen_t))` (this is probably the reason that all of the tests fail on Travis).